modules/RTC/RTCBrowserType.js

```diff
@@ -1,10 +1,10 @@
-const logger = require('jitsi-meet-logger').getLogger(__filename);
```

Comment:
Do you really like more the `import` syntax for this situation or you are changing this only for consistency?

Personally I like the `require` syntax more only for this use case :) But I agree with the consistency argument as well.